### PR TITLE
Add specific instructions on mitm.it when an icon is clicked

### PR DESCRIPTION
**Pull request by Ritiek**
This is a PR for #2313 (2nd point). I used to JS to show content dynamically..

This PR does not add additional OSes/software to the list though (#2313 1st point).

### CODE REVIEW

File: mitmproxy/addons/onboardingapp/templates/index.html [hunk 1]
```diff
@@ -1,47 +1,86 @@
...
+    else if (device == "android") {
+        var text = `<h3>Android: How to install on Android</h3>
+                      <ul>
+                        <li>Open your device's Settings app</li>
+                        <li>Under "Credential storage," tap Install from storage</li>
+                        <li>Under "Open from," tap where you saved the certificate</li>
+                        <li>Tap the file</li>
+                        <li>If prompted, enter the key store password and tap OK</li>
+                        <li>Type a name for the certificate</li>
+                        <li>Pick VPN and apps or Wi-Fi</li>
```
Comment (mhils) [inline mitmproxy/addons/onboardingapp/templates/index.html]: Does it really not matter what the user selects here?

**Comment by Ritiek** [comment]
@mhils Yeah, it does matter. I tested it and it wouldn't work for me if I choose WiFi but works with choosing VPN & Apps. I pushed a commit to fix the line.

File: mitmproxy/addons/onboardingapp/templates/index.html (hunk 1 above)
Comment (Ritiek) [inline mitmproxy/addons/onboardingapp/templates/index.html]: Yeah, it does matter. I tested it and it wouldn't work for me if I choose WiFi but works with choosing VPN & Apps. I pushed a commit to fix the line.

**Comment by Ritiek** [comment]
Sure, I'd love to join the Slack channel. Thanks for building mitmproxy. :blush:

**Pull request merged**